packages/blade/src/components/Input/OTPInput/OTPInput.tsx

```diff
@@ -251,7 +251,7 @@ const OTPInput = ({
   };
 
   return (
-    <>
+    <Box {...dataAttribute('blade-component', 'otp-input')}>
```

Comment:
Added this Box wrapper around the OTPInput because previously the jsx structure was like so: 

```
<>
  <Box /> // main input
  <Box />  // hint content
</>
```
This is problematic since if users tries to wrap the OTP component with their own Flex components the hints & otp's content will be treated as separate flex child.